test/evaluator/Test_Evaluator_Prelude.re

```diff
@@ -3,7 +3,11 @@ open Language;
 
 module UG = Grammar.UnitGrammar;
 
-let dhexp_typ = testable(Fmt.using(Exp.show, Fmt.string), DHExp.fast_equal);
+let dhexp_typ =
+  testable(
+    Fmt.using(Exp.show, Fmt.string),
+    DHExp.fast_equal(~ignore_constructor_types=true) // This is a bad idea to just blanket ignore the type
```

Comment:
Are we good just keeping this for now? I'm not sure where the constructor types are used in dynamics and when it would matter.

Reply:
i didn't know we had types on constructors sooo

Reply:
These tests will fail:

https://github.com/hazelgrove/hazel/pull/1734/files#diff-a5b6e71606758826f0a3f4c4052b3fa0223d021509a0238b4103574a0127d484R209

But we can rewrite those not to use the parser for just the ones that have constructors.